BL7011/file_processing.py

```diff
@@ -5,21 +5,24 @@
 
     Authors: Dayne Sasaki, Damian Günzing
 """
+import os.path
 
 import numpy as np
 from glob import glob
 from os.path import basename
-import nexusformat.nexus
 import pandas as pd
 import h5py
+from IPython.display import display
+from BL7011 import data_processing as dp
+from BL7011 import plotting as plt
```

Comment:
I would suggest using another abbreviation than `plt`, since it is widely used for `matplotlib.pyplot` imports. Maybe just use `pt`?

Reply:
Sounds good; plt was changed to pt in [33a4b58]